teradata_to_exasol.sql

```diff
@@ -468,8 +550,11 @@ STATEMENT 'SELECT ''connection to teradata works''';
 
 -- Finally start the import process
 execute script database_migration.TERADATA_TO_EXASOL(
-    'TERADATA_DB'     -- name of your dataabase connection
-    ,true             -- case sensitivity handling for identifiers -> false: handle them case sensitiv / true: handle them case insensitiv --> recommended: true
-    ,'MIGRATION'     -- schema filter --> '%' to load all schemas except 'DBC' / '%pub%' to load all schemas like '%pub%'
-    ,'%'--'DimCustomer'              -- table filter --> '%' to load all tables
-);
\ No newline at end of file
+    'TERADATA_DB'	-- name of your dataabase connection
+    ,true        	-- case sensitivity handling for identifiers -> false: handle them case sensitiv / true: handle them case insensitiv --> recommended: true
+    ,'RETAIL_2020'	-- schema filter --> '%' to load all schemas except 'DBC' / '%pub%' to load all schemas like '%pub%'
+    ,'%'			--'DimCustomer' -- table filter --> '%' to load all tables
+    ,true			-- boolean flag to create checking tables
```

Comment:
Set to false in the example please. But explain also somewhere what it does (where it creates tables, how many tables - I see in database_migration there is a overview table, whereas in the schema itself it creates multiple tables.
By the way it is a great feature which we should use in the other db migrations, too.